haiku-apps/heidi/heidi-1.0.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="Native IDE for Haiku"
+DESCRIPTION="Heidi is a useful IDE for Haiku where coders can edit, \
+build, run, and debug their code projects all in one window."
+HOMEPAGE="https://github.com/waddlesplash/Heidi/"
+COPYRIGHT="2014 Austin Cavalier"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="ec676d2b45043b7a70d606f5fe2dbf05f3c45ed1"
+SOURCE_URI="https://github.com/waddlesplash/Heidi/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="dca3b020f36e83d0dc83b10b1362b73c6f2240605122fefbfc554e49ad70df8c"
+SOURCE_FILENAME="Heidi-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="Heidi-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86_64"
```

Comment:
scintilla is set as broken on x86_gcc2, maybe you could add a secondary_architecture here?